test/index.test.js

```diff
@@ -241,6 +241,19 @@ describe('config parser', () => {
                             /YAMLException: Cannot have nonexistent job in stages: publish,deploy/
                         );
                     }));
+
+                it('returns an error if a job within a stage is triggered by jobs that are not within the same stage', () =>
+                    parser({
+                        yaml: loadData('pipeline-with-stages-and-invalid-triggers.yaml'),
```

Comment:
Looks like `pipeline-with-stages-and-invalid-triggers.yaml` file got missed out during check-in

Reply:
file is committed now